### PR TITLE
Magento 113

### PR DESCRIPTION
**Pull request by suraiyavic**

### CODE REVIEW

File: Model/DutyCalculator.php [hunk 1]
```diff
@@ -577,11 +577,18 @@ protected function prepareRequest($freightCharge, $shippingAddress)
...
-                $itemData['countryOfOrigin'] = $this->getCountryOfOrigin($item->getSku());
-                if (!$itemData['countryOfOrigin']) {
-                    $itemData['countryOfOrigin'] = $request['senderAddress']['country'];
+
+                //country of origin from uploaded hs code file
+                $countryOfOrigin = $this->getCountryOfOrigin($item->getSku());
+                if (!$countryOfOrigin) {
+                    $countryOfOrigin = $this->reachHelper->getDefaultCountryOfOrigin();
+                }
+                //country of origin from default country of origin drop down
```
Comment (lizzyWithReach) [inline Model/DutyCalculator.php]: small detail, this comment is in a weird place I would have expected it inside of the if statement (if it's needed at all, the code is fairly self-explanatory)

**Review by lizzyWithReach — APPROVED** [review]
small comment note, approving because it's not a big deal but if you would like to amend it then great! If not, feel free merge this PR when you merge the other PRs (because there's code from the other PRs in this one)

File: Model/DutyCalculator.php (hunk 1 above)
Comment (suraiyavic) [reply Model/DutyCalculator.php]: commit 7f6806366f5b4417cbbe8f01d32b4507b9b24e06.
Thanks!

**Review by lizzyWithReach — APPROVED** [review]
Yay! Thank you

**Pull request merged**